scripts/enrich.py

```diff
@@ -39,6 +39,22 @@ def version(self, v):
 SPECIAL_TOKENS = """()[]{}'@#:;"%&`’,.?!/\\^®"""
 
 
+# object to easily report on found records for title search
+class Searched_record:
+    work = None
+    method = ""
+    title_matches = 0
+    good_matches = 0
+    distance = 0
+
+    def __init__(self, work, method, title_matches, good_matches, distance):
+        self.work = work
+        self.method = method
+        self.title_matches = title_matches
+        self.good_matches = good_matches
+        self.distance = distance
```

Comment:
```suggestion
class SearchedRecord:
    def __init__(self, work, method, title_matches, good_matches, distance):
        self.work = work
        self.method = method
        self.title_matches = title_matches
        self.good_matches = good_matches
        self.distance = distance
```
1. These class variables are redundant I think?
Also, I don't know if you know this but i was caught out by how [class variables work in Python](https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables). (TLDR: the value is shared across instances, e.g. changing the value in one instance changes it in all other instances of this class)

2. Class names in Python follow CapWord convention ([PEP 8](https://peps.python.org/pep-0008/#class-names))

Reply:
LGTM